signing_service/requirements.txt

```diff
@@ -1,7 +0,0 @@
-coverage
-mypy
-pytest
-pytest-cov
```

Comment:
Why are you removing requirements? The stuff that's in `setup.py` is not really equivalent to this.

Reply:
Then the removal should be a separate commit. As it is now both this and conversion to `setup.py` are mixed in a single commit and it's not at all clear that this is what happened here so it's impossible to review this change. Please split the commit so that each change as a proper description.

Reply:
Also `setup.py` has hard-coded versions while `requirements` did not. I don't think we should hard-code versions of these dependencies. We should keep the `txt`/`lock` split like in Concent because signing service is not a library and should be running with specific versions.

Reply:
> Then the removal should be a separate commit.

I think it should be the same, if you consider this as a change of the way in which this application is being set up. You can set it up both before and after this commit.

I have added back `txt/lock` and removed requirements from `setup.py`.